spot_driver/launch/spot_driver.launch.py

```diff
@@ -133,15 +131,25 @@ def create_rviz_config(robot_name: str) -> None:
 
 
 def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
+    logger = logging.getLogger("spot_driver_launch")
+
```

Comment:
I think this was unintentionally left here

Reply:
It is required few lines below by the `SpotWrapper`. I tried passing None but it fails at runtime.